projects/ngrx.io/content/guide/store/testing.md

```diff
@@ -1,6 +1,65 @@
 # Testing
 
-### Providing Store for testing
+### Using a Mock Store
+
+The `provideMockStore()` method registers providers that allow you to mock out the `Store` for testing functionality that has a dependency on `Store` without setting up reducers. 
```

Comment:
I think this is not a `method` but just `function`. isn't?

Reply:
Yep, `function` is better.